packaging/installer/methods/macos.md

```diff
@@ -36,6 +37,25 @@ brew install netdata
 
 Skip on ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
 
+## Install Netdata with MacPorts
+
+To install Netdata with MacPorts, simply run:
```

Comment:
Let's add a link to the https://ports.macports.org/port/netdata/ where users can check the Agent version/additional info.